src/components/BlogForm.js

```diff
@@ -27,6 +28,7 @@ class BlogForm extends Component {
   };
   onButtonUpdateEntry = () => {
     const { title, body, entryType, tags, level } = this.state;
+    const userId = this.props.userId;
```

Comment:
Why do you have 2 props in this component: `userId` and `auth.uid`? What's the difference between the two?